### ISSUE TITLE
3.6.1 panic: Static resource is not a valid zip file

### ISSUE THREAD

**Issue by t510599** [cloudreve/Cloudreve#1568]
**Describe the bug**
After upgrading to version 3.6.1, the service failed to start.

docker compose log:
```
cloudreve-cloudreve-1  | [Panic]   2022-12-17 02:41:32 Static resource is not a valid zip file: zip: not a valid zip file
cloudreve-cloudreve-1  | panic: Static resource is not a valid zip file: zip: not a valid zip file
cloudreve-cloudreve-1  |
cloudreve-cloudreve-1  | goroutine 1 [running]:
cloudreve-cloudreve-1  | github.com/cloudreve/Cloudreve/v3/pkg/util.(*Logger).Panic(0x4000421650, {0x11c9788?, 0x0?}, {0x400029fdb0?, 0x119d1f9?, 0xf?})
cloudreve-cloudreve-1  |        /cloudreve_backend/pkg/util/logger.go:74 +0x94
cloudreve-cloudreve-1  | github.com/cloudreve/Cloudreve/v3/bootstrap.NewFS({0x13ba5d8, 0x0})
cloudreve-cloudreve-1  |        /cloudreve_backend/bootstrap/fs.go:17 +0x140
cloudreve-cloudreve-1  | main.init.0()
cloudreve-cloudreve-1  |        /cloudreve_backend/main.go:39 +0x1d8
```

**To Reproduce**
1. use default docker-compose settings in the [document](https://docs.cloudreve.org/getting-started/install#docker-compose)
2. `docker compose up`

**Comment by vvisionnn** [comment]
Hey @t510599 

thanks for your feedback, I have just followed the docs and tested docker compose file, cannot repro this panic, here is the test log
<img width="862" alt="Screenshot 2022-12-17 at 09 43 10" src="https://user-images.githubusercontent.com/24761186/208217557-f5fa9a6e-2091-4389-b9ae-c7b23274dec5.png">
and here is the log I have requested the website (home page / login page) and the static resources can be displayed, seems everything just works fine
<img width="1201" alt="Screenshot 2022-12-17 at 09 44 50" src="https://user-images.githubusercontent.com/24761186/208217720-0e2457c1-f489-48f3-9f03-5c3ee1c1c113.png">

here is the log I have restarted the compose container, the persisted status can be restored
<img width="849" alt="Screenshot 2022-12-17 at 09 45 05" src="https://user-images.githubusercontent.com/24761186/208217784-24b91444-a6e0-4c2e-b272-52bc6ad54b7d.png">

well if you can share more details like what is the version that you have upgraded from, what is the api did you requested, that will help us figure out the real problem

**Comment by t510599** [comment]
Oh sorry, I found that I have not mentioned my system environment.

The current system settings I am using:
- Ubuntu 22.04 kernel 5.15.0-1025-oracle aarch64
- Docker version 20.10.21, build baeda1f
- Docker Compose version v2.12.2
- Image: cloudreve/cloudreve:latest digest `sha256:ebeb49427a1922d57f7dba45bce1a8c693a08dadac2770e8eb82d2b64dec9c47`

I tried again with another newly spinned up vm, and the issue still exists.   
Please check again.  Thank you.

**Comment by Code-wgx** [comment]
openwrt immortalwrt-bcm27xx-bcm2711-rpi-4-ext4-factory

**Comment by Code-wgx** [comment]
I have the same problem
Docker plug-in of openwrt system
Docker version 20.10.17, build 100c701

**Comment by HFO4** [comment]
This issue is due to a timeout while fetching yarn packages. It should be fixed in 3.6.2.
